skopt/utils.py

```diff
@@ -1,9 +1,35 @@
 import numpy as np
 from scipy.optimize import OptimizeResult
 
-def pack_optimize_result(Xi, yi, space, rng, specs, models=None):
+def set_results(Xi, yi, space=None, rng=None, specs=None, models=None):
```

Comment:
at first reading (with first ☕ of the day) `set_results` made me think "ah, they set some properties on the first argument". Would `create_result` be a better name?